### PR TITLE
Swift: update to 2.2 release

### PR DESCRIPTION
**Pull request by alekseysidorov**
### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?
- [ ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Have you built your formula locally prior to submission with `brew install <formula>`?
- [ ] Have you successfully ran `brew tests` with your changes locally?

### CODE REVIEW

**Comment by dunn** [comment]
Missed one of those checkboxes :speak_no_evil: 
```
        Error: 2 problems in 1 formula
swift:
 * 13: Trailing whitespace was found
 * 18: Trailing whitespace was found
```

File: Library/Formula/swift.rb [hunk 1]
```diff
@@ -3,24 +3,24 @@ class Swift < Formula
   homepage "https://github.com/apple/swift"
 
   stable do
-    url "https://github.com/apple/swift/archive/swift-2.2-SNAPSHOT-2016-03-01-a.tar.gz"
-    version "2.2-SNAPSHOT-2016-03-01-a"
-    sha256 "cefd56b08df9ed4867985ce2669b9ca544b8fbda75b0e8932047a2474d7db209"
+    url "https://github.com/apple/swift/archive/swift-2.2-RELEASE.tar.gz"
+    version "2.2-RELEASE"
```
Comment (dunn) [inline Library/Formula/swift.rb]: This can just be `2.2`.

**Comment by alekseysidorov** [comment]
Request updated. I hope that everything is fine.

**Comment by dunn** [comment]
Looks great, thanks! Merged as 86f6a40cceb4074806d0b5a8581a2d45a49b527d.

**Pull request closed without merging**